android/mobile/gradle/libs.versions.toml

```diff
@@ -0,0 +1,27 @@
+[versions]
+agp = "8.4.0-beta02"
+kotlin = "1.9.0"
+coreKtx = "1.12.0"
+lifecycleRuntimeKtx = "2.7.0"
+activityCompose = "1.8.2"
+composeBom = "2024.03.00"
+openpass = "2.1.0"
```

Comment:
Q: Does the repo have renovate configured in it? I'd strongly recommend, as many of these are already behind (e.g. kotlin).

Also to make sure it doesn't get too stale, I'd add a build step for the sample app in CI.

Reply:
It doesn't, nor does the SDK. I will discuss with @christriddle-thetradedesk about configuring this separately across all the repos that contain Android sources.